### ISSUE TITLE
Bug: Setting custom "density" in settings.json does not work

### ISSUE THREAD

**Issue by ulilu** [sigboost-inc/midiglue-forum#7]
I'm on a Mac, running MacOS 10.14.6 (Mojave). 

I add a settigns.json file to the directory `user/<username>/Documents/midiglue` and set a different density, as described in the manual.

Then I restart the Midiglue editor, but the density is still the same.

It seems like the settings.json is not read or interpreted by the Midiglue Editor ?

**Comment by kuguma** [comment]
First of all, I'd like to check if the setting.json is in the right place.
For example, if the user name is John, then the file must exist at the following location:

`/Users/John/Documents/midiglue/settings.json`

Also, if the json file is corrupted, the settings will not be reflected. Please check together.

**Comment by ulilu** [comment]
Yes, the file is placed in /Users/uli/Documents/midiglue/settings.json and is a readable json-file.

**Comment by kuguma** [comment]
We have not been able to reproduce the problem yet. It could be due to Mojave, but we no longer have Mojave ...

1. Is it possible to paste the json contents here?
1. Also, is the character code UTF-8?

**Comment by kuguma** [comment]
I found that the old documentation was incorrect. The correct json file does not have a "," at the end.
Please check here. https://sigboost.audio/midiglue/manual/editor/#_13
The wrong json file will not be read.
